### DISCUSSION TITLE
Size exclusion chromatography

### DISCUSSION THREAD

**Discussion by glnegri** [vdemichev/DiaNN#958] [General]
What would be a good search strategy and the best parameters to do a library-free search and quantification of a 20 fractions SEC experiment for which I also have whole cell runs (no separation)? Should I include SEC and whole runs together or run them separately? Library generation mode? Cross-run normalization off?

**Comment by vdemichev** [comment]
Cross-run normalisation off, library generation set to IDs, RT and IM profiling, run SEC runs independently from bulk. 
Running SEC together with bulk will likely increase the IDs numbers in the bulk - if that's the purpose, can do that separately. But this might affect SEC quantities slightly, so better to do SEC quantitation without bulk sample included.

**Comment by grandrea** [comment]
I am performing a similar experiment- would you suggest also switching off MBR, since I am loading different quantities across the different fractions? As an aside, It would be nice to be able to define fractions and MBR only across same fractions between replicates or + or - 1 fraction...

**Reply by vdemichev** [reply]
No, MBR should be on unless you are using a DIA-based empirical lib, in almost all cases.

**Reply by grandrea** [reply]
thanks so much for the lightning reply!
